### PR TITLE
[dv, chip] Enable UNR flow

### PR DESCRIPTION
**Pull request by sriyerg**
This sets up the UNR flow for the chip. Though, it does not work due to
errors thrown by the UNR tool. I am working with SNPS to resolve that.

Signed-off-by: Srikrishna Iyer <sriyer@google.com>

### CODE REVIEW

**Comment by cindychip** [comment]
Hey Sri, thanks for working on that. I am concerning that for chip level it might take a very long time to run UNR. For otp_ctrl, I ran UNR for almost two days to get a result.

**Comment by sriyerg** [comment]
> Hey Sri, thanks for working on that. I am concerning that for chip level it might take a very long time to run UNR. For otp_ctrl, I ran UNR for almost two days to get a result.

Ohh wow. I didn't imagine it would take that long. Given that I am not spending too much time on it, perhaps we can kick it off (when the flow is fixed) and see how it goes...If it takes over 2 weeks then its probably not worth it.

**Comment by sriyerg** [comment]
> > Hey Sri, thanks for working on that. I am concerning that for chip level it might take a very long time to run UNR. For otp_ctrl, I ran UNR for almost two days to get a result.
> 
> Ohh wow. I didn't imagine it would take that long. Given that I am not spending too much time on it, perhaps we can kick it off (when the flow is fixed) and see how it goes...If it takes over 2 weeks then its probably not worth it.

BTW, at the chip level, we are only covering toggles on DUT ports and some glue logic, so the state space is much smaller. I am hoping its not too bad.

**Comment by cindychip** [comment]
Oh I see, toggles should be faster than a full UNR!

**Comment by weicaiyang** [comment]
> Hey Sri, thanks for working on that. I am concerning that for chip level it might take a very long time to run UNR. For otp_ctrl, I ran UNR for almost two days to get a result.

That's too long. If you only ran with coverage from one test, perhaps you can run with nightly regression coverage next time. With less uncovered database, UNR can probably run faster.

File: hw/top_earlgrey/dv/cov/unr.cfg [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+-clock IOC6 100
+
+# Provide the reset specification: signal_name, active_value, num clk cycles reset to be active.
+-reset POR_N 0 20
+
+# Black box some of the modules
+# -blackBoxes -type design *
+
+# Include common el file, so that it doesn't generate reviewed common exclusions
+-covEL $dv_root/tools/vcs/common_cov_excl.el
```
Comment (weicaiyang) [inline hw/top_earlgrey/dv/cov/unr.cfg:18]: looks like common_cov_excl doesn't work with chip-level

**Review by weicaiyang — APPROVED** [review]
LGTM

**Pull request merged**